src/pages/settings/InitialPage.js

```diff
@@ -0,0 +1,163 @@
+        avatar: PropTypes.string,
+    }),
+
+    // Information about the network
+    network: PropTypes.shape({
+        // Is the network currently offline or not
+        isOffline: PropTypes.bool,
+    }),
+
+    // The session of the logged in person
+    session: PropTypes.shape({
+        // Email of the logged in person
+        email: PropTypes.string,
+    }),
+};
+
+const defaultProps = {
+    myPersonalDetails: {},
+    network: null,
+    session: {},
+};
+
+const menuItems = [
+    {
+        title: 'Profile',
+        icon: Profile,
+        route: 'profile',
+    },
+    {
+        title: 'Preferences',
+        icon: Gear,
+        route: 'preferences',
+    },
+    {
+        title: 'Change Password',
+        icon: Lock,
+        route: 'password',
+    },
+    {
+        title: 'Payments',
+        icon: Wallet,
+        route: 'payments',
+    },
+];
+
+const InitialSettingsPage = ({
+    myPersonalDetails,
+    network,
+    session,
+}) => {
+    // On the very first sign in or after clearing storage these
+    // details will not be present on the first render so we'll just
+    // return nothing for now.
+    if (_.isEmpty(myPersonalDetails)) {
+        return null;
+    }
+    return (
+        <>
+            <HeaderGap />
+            <HeaderWithCloseButton title="Settings" onCloseButtonPress={redirectToLastReport} />
+            <View
+                pointerEvents="box-none"
+                style={[
+                    styles.settingsPageBackground,
+                ]}
+            >
+                <View style={styles.w100}>
+                    <View style={styles.settingsWrapper}>
+
+                        <View style={[styles.mb3]}>
+                            <AvatarWithIndicator
+                                size="large"
+                                source={myPersonalDetails.avatar}
+                                isActive={network && !network.isOffline}
+                            />
+                        </View>
+                        <Text style={[styles.settingsDisplayName, styles.mt1]} numberOfLines={1}>
+                            {myPersonalDetails.displayName
+                                ? myPersonalDetails.displayName
+                                : Str.removeSMSDomain(session.email)}
+                        </Text>
+                        {myPersonalDetails.displayName && (
+                        <Text style={[styles.settingsLoginName, styles.mt1]} numberOfLines={1}>
+                            {Str.removeSMSDomain(session.email)}
+                        </Text>
+                        )}
+                    </View>
+                    {menuItems.map(item => (
+                        <MenuItem
+                            key={item.title}
+                            title={item.title}
+                            icon={item.icon}
+                            onPress={() => redirect(ROUTES.getSettingsRoute(item.route))}
```

Comment:
NAB, this is more of a question for @NikkiWines. I am curious how useful it is to have a `getSettingsRoute()` when there is a set number of routes. Would it be better to just have things like `ROUTES.SETTINGS_PASSWORD: '/settings/password'`

Reply:
We definitely could do `ROUTES.SETTINGS_PASSWORD: '/settings/password'` and have one route for each Settings sub-route but I find it just as readable to do `getSettingsRoute()` which accomplishes the same thing but with fewer lines. 

Ultimately happy to go with either.

Reply:
Cool. My main reasoning here is that they are really more like constants and not something like `getReportRoute()` which takes in unlimited variations of `reportID`.

Reply:
@NikkiWines @marcaaron another benefit would be that you'd easily know what sub-routes are available in the `ROUTES` file without having to dig into the code

Reply:
> something like getReportRoute() which takes in unlimited variations of reportID.

Yeah `getReportsRoute()` - as well as the now removed `getSigninWithExitToRoute()` - were my models for how to route to subpages originally. However I agree that it's unlikely we'll have so many settings sub pages that we'd _need_ to rely on something like `getSettingsRoute()`

> another benefit would be that you'd easily know what sub-routes are available in the ROUTES file without having to dig into the code

That's a good point as well - @Maftalion feel free to update this to the explicitly defined route instead of using `getSettingsRoute()`